### DISCUSSION TITLE
LaTeX widget

### DISCUSSION THREAD

**Discussion by bcfoltz** [biolab/orange3#6686] [Ideas]
It would be nice to have a widget in which you could paste some LaTeX and then have it display all nice and spiffy. Of course, a bigger feature would be a LaTeX editor, but just having something for display would be a nice addition.

**Comment by markotoplak** [comment]
Thanks. How would you use it though? Can you describe an imaginary workflow where the suggested widget would be useful?

**Reply by bcfoltz** [reply]
Hello! This is similar to my Markdown idea. While the text tool and the arrow tool offer some rudimentary annotations, I still find that there is room for more expository features in Orange that would have high value, especially in an educational context.

I use Orange to teach and I find that I am frequently wanting to show how some of the numbers are derived and add additional information and context. For example, I was doing some Decision Tree lessons, and when I was explaining the Gini Impurity I wanted to put the equations in the workflow since they are relatively simple without having to leave Orange. Same thing with some of the confusion matrix calculations, linear equations, etc. To get even fancier, the LaTex could also be dynamic, inserting values into the LaTex using variables (but that might be a later addition).

Of course, because of the flexibility of LaTeX, it would not have to be just numbers and equations. It's a solid base to add new features and put some magic sprinkles on existing Orange widgets, tools, and outputs.

Just a few thoughts! Thank you for your time.
